.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
                   python-version: ${{ matrix.python-version }}
             - run: |
                 sudo apt update
-                sudo apt install graphviz imagemagick x11-apps xserver-xephyr xterm xvfb libpulse-dev lm-sensors
+                sudo apt install libdbus-1-dev graphviz imagemagick x11-apps xserver-xephyr xterm xvfb libpulse-dev lm-sensors
```

Comment:
Why this change?

Reply:
we weren't statically analyzing dbus stuff before because missing library, now we are!

Reply:
:tada: 

please make it a separate commit with that message so that we can remember why we have this

Reply:
:+1: